cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -150,6 +185,26 @@ def token_body_with_valid_expiration(token_body)
     end
     token_body
   end
+
+  def base64_x5t_from_certificate(cert)
+    cert_thumbprint = OpenSSL::Digest::SHA1.hexdigest(cert.to_der)
+    Base64.urlsafe_encode64(cert_thumbprint)
```

Comment:
```suggestion
    Base64.urlsafe_encode64(cert_thumbprint, false)
```
> Note that the result can still contain ‘=’. You can remove the padding by setting padding as false.

From there https://apidock.com/ruby/Base64/urlsafe_encode64

> The "x5t" (X.509 certificate SHA-1 thumbprint) Header Parameter is a base64url-encoded SHA-1 thumbprint (a.k.a. digest) of the DER encoding of the X.509 certificate [RFC5280] corresponding to the key used to digitally sign the JWS.

From there https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.7

base64url-encoded string should not contain `=` signs.